tests/ignite/distributed/utils/__init__.py

```diff
@@ -228,6 +228,32 @@ def _test_distrib_barrier(device):
     assert tt.item() == true_res + 10.0
 
 
+def _test_distrib_new_group(device):
+    from ignite.distributed.comp_models import _SerialModel
+
+    bnd = idist.backend()
+    if idist.get_world_size() > 1:
+        ranks = [0, 1]
+        if idist.has_native_dist_support and bnd in ("nccl", "gloo", "mpi"):
+
+            assert idist.new_group(ranks).rank == dist.new_group(ranks=ranks).rank
+        elif idist.has_xla_support and bnd in ("xla-tpu"):
+
+            assert idist.new_group(ranks) == [ranks]
+        elif idist.has_hvd_support and bnd in ("horovod"):
+            from horovod.common.process_sets import ProcessSet
+
+            assert idist.new_group(ranks).rank == ProcessSet(ranks).rank
+
```

Comment:
backend is none and
else (idist.get_world_size() == 1) :

Reply:
I added as below but it occurs errors because backend is not `None`
And should the `test` take the argument `model` to ensure the model is `Serial` in `else` clause?

```python
else:
        assert bnd is None
        assert idist.new_group(ranks) == ranks
```

Reply:
So, what it is the backend ?
No need to add new args, try to figure out how to do what we want to test here.